region/brinstar/kraid.json

```diff
@@ -767,7 +767,13 @@
                 {
                   "name": "Base",
                   "notable": false,
-                  "requires": []
+                  "requires": [
+                    {"or": [
+                      "HiJump",
+                      "h_canFly",
+                      "canWalljump"
```

Comment:
SpringBall Jump works here. So does a CrouchJump+DownGrab, and a 16-frame shinespark if coming in charged from node 2 with ~150 frames remaining.

Reply:
Added the crouch jump down grab. I don't think the others are necessary.